fdbserver/DataDistribution.actor.cpp

```diff
@@ -1626,6 +1627,28 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 		return std::pair<Reference<TCMachineTeamInfo>, int>(retMT, minNumProcessTeams);
 	}
 
+	// Find the server team whose members are on the most number of server teams
+	std::pair<Reference<TCTeamInfo>, int> getServerTeamWithMostProcessTeams(bool chooseUnhealthyTeam) {
+		Reference<TCTeamInfo> retST;
+		int maxNumProcessTeams = 0;
+
+		for (auto& t : teams) {
+			if (chooseUnhealthyTeam && t->isHealthy()) {
+				continue;
+			}
+			int numProcessTeams = 0;
+			for (auto& server : t->getServers()) {
+				numProcessTeams += server->teams.size();
+			}
+			if (numProcessTeams > maxNumProcessTeams) {
+				maxNumProcessTeams = numProcessTeams;
+				retST = t;
+			}
+		}
+
+		return std::pair<Reference<TCTeamInfo>, int>(retST, maxNumProcessTeams);
```

Comment:
Small tip. You can avoid the need to specify template arguments by using, `std::make_pair(retST, maxNumProcessTeams)`.

Reply:
Ah, I prefer the explicit type. :)